### PR TITLE
add interface EvmConfigParams struct, update BlockchainConfiguration

### PR DESCRIPTION
**Pull request by Mr-Leshiy**

### CODE REVIEW

File: jormungandr/Cargo.toml [hunk 1]
```diff
@@ -92,3 +92,4 @@ soak-test = []
 systemd = ["tracing-journald"]
 gelf = ["tracing-gelf"]
 prometheus-metrics = ["prometheus"]
+evm = []
```
Comment (Zeegomo) [inline jormungandr/Cargo.toml:95]: I think this also relies on `chaim-impl-mockchain/evm`

File: Cargo.lock [hunk 2]
```diff
@@ -33,6 +33,12 @@ version = "1.2.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
```
[the remark this answers is not in the document]
Comment (Mr-Leshiy) [reply Cargo.lock:33]: I have update to the latest `chain-libs`, I assume that is the reason

File: jormungandr-lib/src/interfaces/evm_params.rs [hunk 3]
```diff
@@ -0,0 +1,46 @@
+use chain_impl_mockchain::config;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize, Default)]
+pub struct EvmConfigParams {
+    #[serde(skip)]
```
[the remark this answers is not in the document]
Comment (Mr-Leshiy) [reply jormungandr-lib/src/interfaces/evm_params.rs:6]: I will add full `serde` implementation for this struct after `EvmConfigParams` on the `chain-libs` side will be updated, actually this PR.
Intention to derive Serialize/Deserialize was to have the necessary functionality and preliminary add tests and correct usage in the `BlockchainConfiguration`.

**Comment by Mr-Leshiy** [comment]
need to merge first https://github.com/input-output-hk/chain-libs/pull/729

**Review by eugene-babichenko — CHANGES_REQUESTED** [review]
The tests are needed to be fixed, otherwise looking good, deps merged.

**Review by saibatizoku — APPROVED** [review]
Looks good to me, much cleaner, thanks.

**Review by eugene-babichenko — APPROVED** [review]

**Pull request merged**